### DISCUSSION TITLE
OvenStreamEncoder connection fails

### DISCUSSION THREAD

**Discussion by deliaam** [OvenMediaLabs/OvenMediaEngine#1144] [Help]
Hi, I have started the ome image in my PC but when I try to stream from the OvenStreamEncoder app (or any other app with rtmp stream) I get "Connection failed. Please check your input URL". When I use OBS in my PC everything works right. Of course, I am connected to the same Wi-Fi connection in both my smartphone and PC.
This is the URL I am using in OBS and in OvenStreamEncoder :
![image](https://user-images.githubusercontent.com/72461334/226325075-48aab40d-9f90-43c3-a742-fdf8fa81377a.png)

**Comment by getroot** [comment]
Did you check your firewall?

**Reply by deliaam** [reply]
Firewall is deactivated

**Reply by getroot** [reply]
If both the PC's firewall (OME is running in Docker on Windows?) or the WIFI device's firewall are deactivated... I can't seem to guess anything further. In my environment it works fine.

**Reply by getroot** [reply]
Oh, you're the one who opened the WSL discussion a while ago, right? Has the WSL issue been resolved?

**Comment by getroot** [comment]
Did you solve this problem? Could you share what the problem was and how you solved it for the betterment of the community?
